Source/Core/Core/IOS/USB/Bluetooth/BTReal.cpp

```diff
@@ -547,7 +559,8 @@ bool BluetoothReal::OpenDevice(libusb_device* device)
   const int ret = libusb_open(m_device, &m_handle);
   if (ret != 0)
   {
-    PanicAlertT("Failed to open Bluetooth device: %s", libusb_error_name(ret));
+    m_last_open_error = fmt::format(Common::GetStringT("Failed to open Bluetooth device: %s"),
```

Comment:
Does `fmt::format` support `%s`-style formatting? All the other places where we use it use `{}`.

Reply:
...oh no, I forgot to change the `%s`s. Thanks for catching this!

Reply:
It does support them to a degree, it mentions it under the `printf` style formatting: https://fmt.dev/latest/api.html#printf-api but whether you have to include the other header, not sure.